ui/js/controllers/perf/compare.js

```diff
@@ -413,15 +413,14 @@ perf.controller('CompareSubtestResultsCtrl', [
             $scope.compareResults = {};
             $scope.titles = {};
 
-            $scope.subtestTitle = ($scope.platformList[0].split(' ')[0] + " " +
-                                   $scope.testList[0].split(' ')[0]);
-            window.document.title = $scope.subtestTitle + " subtest comparison";
```

Comment:
we are removing 'subtest comparison', is it clear from the rest of the page that we are looking at subtests?